axisregistry/Lib/axisregistry/data/x_rotation.textproto

```diff
@@ -0,0 +1,14 @@
+#Rotation in X, based on https://github.com/andyclymer/Tilt-Typeface
+tag: "XROT"
+display_name: "Rotation in X"
+min_value: -180
+default_value: 0
+max_value: 180
+precision: 0
+fallback {
+  name: "Default"
+  value: 0
+}
+fallback_only: false
+description:
+  "Letters rotate around the X axis."
```

Comment:
Suggest that we update the description to change the word 'Letters' to something more generic.  'Shapes'?  Let's merge so that we can test and address it in the googlefonts/axisregistry repo.  My fault.  I missed this on the review on that side.

Reply:
https://github.com/googlefonts/axisregistry/issues/92